exercises/twelve-days/test/Tests.hs

```diff
@@ -0,0 +1,146 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import TwelveDays (recite)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "responseFor" $ for_ cases test
+  where
+      test Case{..} = it description assertion
+        where
+          assertion = recite start stop `shouldBe` expected
+
+data Case = Case { description :: String
+                 , start       :: Int
+                 , stop        :: Int
+                 , expected    :: [String]
+                 }
+
+cases :: [Case]
+cases = [ Case { description = "first day a partridge in a pear tree"
+               , start = 1
+               , stop = 1
+               , expected = [ 
+                  "On the first day of Christmas my true love gave to me, a Partridge in a Pear Tree."
+                 ]
+               }
+         , Case { description = "second day two turtle doves"
```

Comment:
could I ask you to check the alignment of this line and all other `Case` lines for me? My eyes may deceive me but I was thinking they are not aligned right now.

Reply:
Interesting. Yea, I can totally do that. Somehow I managed to mis-align them 0.o